assets/pesy-package.template.json

```diff
@@ -11,8 +11,12 @@
   "buildDirs": {
     "test": {
       "require": ["<PUBLIC_LIB_NAME>", "rely.lib"],
+      "flags": ["-linkall", "-g"]
+    },
+    "testExe": {
+      "require": ["<TEST_LIB_NAME>"],
       "bin": {
-        "Test<PACKAGE_NAME_UPPER_CAMEL>.exe": "Test<PACKAGE_NAME_UPPER_CAMEL>.re"
+        "RunTests.exe": "RunTests.re"
```

Comment:
I think the names of public binaries should have the package name prefixed because they all get mixed together for all dependencies in some environments.

Reply:
Run<PACKAGE_NAME_UPPER_CAMEL>Tests.exe?

Reply:
Yeah.